bin/pgadmin4/start-pgadmin4.sh

```diff
@@ -17,7 +17,13 @@ source /opt/cpm/bin/common_lib.sh
 enable_debugging
 
 export PATH=$PATH:/usr/pgsql-*/bin
-PGADMIN_DIR='/usr/lib/python2.7/site-packages/pgadmin4-web'
+PGADMIN_DIR=/usr/lib/python3.6/site-packages/pgadmin4-web
+
+if ! command -v python3 &> /dev/null
+then
+  PGADMIN_DIR=/usr/lib/python2.7/site-packages/pgadmin4-web
+fi
```

Comment:
Not a blocker; I'd like this more if it looked like the other checks when I squint:

```suggestion
if ! command -v python3 &> /dev/null
then
  PGADMIN_DIR=/usr/lib/python2.7/site-packages/pgadmin4-web
else
  PGADMIN_DIR=/usr/lib/python3.6/site-packages/pgadmin4-web
fi
```

Reply:
Mooted.